### PR TITLE
Improve README for Windows users

### PR DESCRIPTION
**Pull request by voltagex**
As discussed in #226. First pass, happy to update/move/reword this as needed.

### CODE REVIEW

**Comment by voltagex** [comment]
You might want to exclude README.md changes from CI - I didn't realise how many builds this would trigger!

**Review by warmwaffles — APPROVED** [review]
This is adequate enough to get started.

**Comment by warmwaffles** [comment]
Heh, it's okay the runs are fine. The failing windows builds is a known issue and should be getting fixed upstream soon.
